### PR TITLE
updated image segmentation data prep example. Runs end-to-end with do…

### PR DESCRIPTION
**Pull request by ChipRobie**
…wnloadable data.

### CODE REVIEW

**Comment by ChipRobie** [comment]
Final validated version of Using SAS DLPy to Create Training Data Sets for Image Segmentation Models example with downloadable data.

**Comment by ChipRobie** [comment]
Updated the Comprehensive Image Classification example from quick_start to include more detailed code comments about deploying a saved .sashdat model, per GitHub issue 339. Also made displaying the model1 network architecture an optional bit of code.

Meant for this to be a separate pull request from the image segmentation data prep example update, but the two PRs seem to have been combined. Apologies...

File: examples/segmentation/image_segmentation_data_preparation.ipynb [hunk 1]
```diff
@@ -349,7 +349,7 @@
    "source": [
     "Each input image in `raw` uses a 256 x 256 matrix of pixel _BGR_ values to define a visual picture. Each corresponding ground truth image in `mask` uses a 256 x 256 matrix of pixel _object class_ values, resulting in a pixel-by-pixel object class map. The number of classes required for an image segmentation ground truth map is equal to the number of object classes plus 1. \n",
     "\n",
-    "The image segmentation model in this example detects two object classes in input images: _player_ and _ball_. Every input image pixel BGR array corresponds to a matching ground truth pixel-wise object class array. Each ground truth image pixel contains a class value: it must be 0 (background), or 1 (player), or 2 (ball). \n",
+    "The image segmentation model in this example detects two object classes in input images: _player_ and _ball_. Every input image pixel BGR array corresponds to a matching ground truth pixel-wise object class array. Each ground truth image pixel contains a class value: it must be 0 (background), or 1 (player), or 2 (background). \n",
```
Comment (dxq77dxq) [inline examples/segmentation/image_segmentation_data_preparation.ipynb:352]: Class value 2 should be ball instead of background.

**Review by dxq77dxq — CHANGES_REQUESTED** [review]

File: examples/segmentation/image_segmentation_data_preparation.ipynb (hunk 1 above)
Comment (ChipRobie) [reply examples/segmentation/image_segmentation_data_preparation.ipynb]: Class value 2 typo in image segmentation data prep example updated to 'ball'.

**Comment by ChipRobie** [comment]
PrepareObjectDetectionTable example validated and run end-to-end using downloadable data.

**Review by dxq77dxq — APPROVED** [review]

**Pull request merged**